logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -177,4 +176,4 @@ if __name__ == "__main__":
     rospy.loginfo("Initialized logitech_r400 node")
 
     with LogitechR400Usb() as pointer:
-        pointer.read_data()
+        pointer._read_data()
```

Comment:
If read_data is used in the main, it means it is a public method. Therefore it should not have a leading underscore.